app/Main.hs

```diff
@@ -140,6 +148,7 @@ main = do
     Sources                    -> Spago.sources
     Build                      -> Spago.build
     Test modName               -> Spago.test modName
+    Repl                       -> Spago.repl []
```

Comment:
It should be possible to pass options to the `purs repl` command (like `spago repl --port 3200`)

Reply:
Awesome! We could look at how `psc-package` does it, and use something similar to their [`passthroughArgs`](https://github.com/purescript/psc-package/blob/648da70ae9b7ed48216ed03f930c1a6e8e902c0e/app/Main.hs#L651) combinator for gathering the args, and then handle them generically as they do in [`exec`](https://github.com/purescript/psc-package/blob/648da70ae9b7ed48216ed03f930c1a6e8e902c0e/app/Main.hs#L400)

Reply:
@vyorkin I went on and implemented the passthrough in #66, so once that's merged you can rebase on master and reuse the `passthroughArgs` option to pass args to `purs`, so e.g. this would work:
```bash
spago repl -- --port 3200
```

Reply:
thanks! sorry for delay, I hope I'll get to it today evening